README.md

```diff
@@ -56,3 +59,6 @@ the Arrow-specific specifications (Arrow layout and extension type metadata).
 * [geoarrow-rs](https://github.com/geoarrow/geoarrow-rs/): Rust implementation of the
   GeoArrow specification and bindings to GeoRust algorithms for efficient spatial
   operations on GeoArrow memory. Includes JavaScript (WebAssembly) bindings.
+* [geoarrow-python](https://github.com/geoarrow/geoarrow-python): Bindings to geoarrow-c
```

Comment:
Maybe redundant but starting the description with `Python` probably adds clarity
```suggestion
* [geoarrow-python](https://github.com/geoarrow/geoarrow-python): Python bindings to geoarrow-c
```